[2022-04-03 18:46:02,790] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: workflow1.create_table2 manual__2022-04-03T16:39:59.606063+00:00 [queued]>
[2022-04-03 18:46:02,794] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: workflow1.create_table2 manual__2022-04-03T16:39:59.606063+00:00 [queued]>
[2022-04-03 18:46:02,794] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 18:46:02,794] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-04-03 18:46:02,794] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 18:46:02,798] {taskinstance.py:1264} INFO - Executing <Task(PostgresOperator): create_table2> on 2022-04-03 16:39:59.606063+00:00
[2022-04-03 18:46:02,803] {standard_task_runner.py:52} INFO - Started process 63046 to run task
[2022-04-03 18:46:02,807] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'workflow1', 'create_table2', 'manual__2022-04-03T16:39:59.606063+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/extract_file.py', '--cfg-path', '/var/folders/z6/rr0r2zzn64qd2w2102l7_w3r0000gn/T/tmpc8sdlryq', '--error-file', '/var/folders/z6/rr0r2zzn64qd2w2102l7_w3r0000gn/T/tmpsuote21p']
[2022-04-03 18:46:02,809] {standard_task_runner.py:77} INFO - Job 15: Subtask create_table2
[2022-04-03 18:46:07,838] {logging_mixin.py:109} INFO - Running <TaskInstance: workflow1.create_table2 manual__2022-04-03T16:39:59.606063+00:00 [running]> on host Manvis-MacBook-Air.local
[2022-04-03 18:46:12,886] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=workflow1
AIRFLOW_CTX_TASK_ID=create_table2
AIRFLOW_CTX_EXECUTION_DATE=2022-04-03T16:39:59.606063+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-03T16:39:59.606063+00:00
[2022-04-03 18:46:12,894] {base.py:70} INFO - Using connection to: id: postgres_db. Host: 127.0.0.1, Port: 5432, Schema: airflow_db, Login: postgres, Password: ***, extra: {}
[2022-04-03 18:46:12,945] {dbapi.py:225} INFO - Running statement: CREATE TABLE IF NOT EXISTS prices (id bigint primary key,productid int,pricecomponentid int,price decimal(38,10), unit varchar(255),valid_from date,valid_until date, modificationdate date), parameters: None
[2022-04-03 18:46:12,946] {postgres.py:94} INFO - NOTICE:  relation "prices" already exists, skipping

[2022-04-03 18:46:12,950] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=workflow1, task_id=create_table2, execution_date=20220403T163959, start_date=20220403T164602, end_date=20220403T164612
[2022-04-03 18:46:12,989] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-03 18:46:13,001] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
